GameData/Mods/Archived/Super_Spyglass_20210813.gamedata

```diff
@@ -1,21 +1,25 @@
 {
   "Application": "ark",
-  "Version": "1.0.0",
-  "Created": "2021-06-07T20:10:00.0000000Z",
+  "Version": "1.0.1",
+  "Created": "2021-12-12T18:00:00.0000000Z",
+  "Mod": "[793605978]",
```

Comment:
found some []

Reply:
This one is archived and the correct one is in the Mod Folder.

Reply:
This on had a date in the file name so I just left the one without a date.